app/helpers/mustache_helper.rb

```diff
@@ -394,6 +394,23 @@ def content
     end
   end
 
+  def collection_filter_options
+    ops = Collection.published.select { |c| c.landing_page.present? && c.landing_page.published? }.map do |collection|
+      val = collection_path(collection).split('/').pop
+      {
+        value: val,
+        label: collection.landing_page.title,
+        selected: params['theme'] == val
+      }
+    end
+    {
+      options: ops.unshift({
+        value: '*',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.